geode-docs/tools_modules/gfsh/command-pages/connect.html.md.erb

```diff
@@ -185,7 +182,7 @@ Successfully connected to: [host=GemFireStymon, port=1099]
 **Example of connecting to a remote locator over HTTP:**
 
 ``` pre
-gfsh>connect --use-http=true --url="http://myLocatorHost.example.com:8080/gemfire/v1"
+gfsh>connect --url="http://myLocatorHost.example.com:8080/gemfire/v1"
```

Comment:
This might be more of a @davebarnes97 question, but since `http://<host>:<port>/geode-mgmt/v1` is used as the example in gfsh, should docs use something more similar to `http://myLocatorHost.example.com:8080/geode-mgmt/v1` (remove the `gemfire` part)

Reply:
I agree, I think the example in code and documentation should be aligned, Im going to modify it.